terminal/command.go

```diff
@@ -578,37 +587,52 @@ func breakpoints(t *Term, args string) error {
 	return nil
 }
 
-func setBreakpoint(t *Term, tracepoint bool, argstr string) error {
-	args := strings.Split(argstr, " ")
-	if len(args) < 1 {
-		return fmt.Errorf("address required, specify either a function name or <file:line>")
+func validBreakpointName(name string) bool {
```

Comment:
This check is to figure out where to split `break`'s arguments, the other is a true validation. I have moved the validation logic in `service/api`. If you want I can change `terminal.setBreakpoint` to first pass its whole argument string to the server and if there is an error do the split.